packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -426,6 +426,14 @@ const double _kDefaultFontSize = 14.0;
 ///
 /// ### Inconsistent platform fonts
 ///
+/// By default, fonts differ depending on the platform.
+///
+///  * The default font-family for `Android`,`Fuchsia` and `Linux` is `Roboto`.
+///  * The default font-family for `iOS` is `.SF UI Display`/`.SF UI Text`.
+///  * The default font-family for `MacOS` is `.AppleSystemUIFont`.
+///  * The default font-family for `Windows` is `Segoe UI`.
+// The implementation of differing fonts(depending on the platform) can be found in /packages/flutter/lib/src/material/typography.dart
```

Comment:
I would add a separator line // between the two comment blocks. I don't think we need the '(depending on the platform)' part.  How about something like:

```dart
/// * The default font-family for `Windows` is `Segoe UI`.
//
// The implementation of these defaults can be found in:
// /packages/flutter/lib/src/material/typography.dart'
```

Reply:
@darrenaustin done.